packages/asl-service/lib/middleware/flash-middleware.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function(req, res, next) {
+  if (!req.session.flash) req.session.flash = {};
+
+  // Set a flash message
+  res.flash = (key, value) => {
```

Comment:
I'd much rather see an API that guided usage here. The scope is there is only ever one message so this should be something like.

```suggestion
  res.setFlash = (title, body, type = 'success') => req.session.flash = { title, body, type };
```